server/neptune/workflows/internal/terraform/runners/job_runner.go

```diff
@@ -0,0 +1,61 @@
+package runners
+
+import (
+	"strings"
+
+	"github.com/runatlantis/atlantis/server/neptune/workflows/internal/steps"
+)
+
+// StepRunner runs custom run steps.
+type StepRunner interface {
+	Run(executionContext steps.ExecutionContext, rootInstance *steps.RootInstance, step steps.Step) (string, error)
+}
+
+// JobRunner runs a deploy plan/apply job
+type JobRunner interface {
+	Run(ctx steps.ExecutionContext, job steps.Job, rootInstance *steps.RootInstance) (string, error)
+}
+
+func NewJobRunner(runStepRunner StepRunner, envStepRunner StepRunner) *jobRunner {
```

Comment:
This reads as `runners.NewJobRunner()` which isn't great.  Think about what these objects/interactions actually mean.  I'd prefer like `job.NewRunner()` or something like that.

Reply:
Yeah that makes sense, I have reorganized the runners package to job which has sub packages for different steps (env and run for now)